sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -68,6 +70,8 @@ batch of messages, or iterate over the receiver to receive continuously.
 
 In v7, users should initialize the client via `ServiceBusClient.get_queue_receiver`.  Single-batch-receive
 has been renamed to `receive`, iterating over the receiver for continual message consumption has not changed.
+It should also be noted that if a session receiver is desired, to use the `get_<queue/subscription>_session_receiver`
+function.
```

Comment:
we should mention this in changelog as well.
You could either do in this PR or the session refactor PR.